src/components/A.js

```diff
@@ -1,9 +1,10 @@
 import React, { PropTypes } from 'react'
 import EmailPropTypes from '../PropTypes'
+import includeDataProps from '../includeDataProp'
```

Comment:
Small nit here: could you please make the filename of the module match the import name, `includeDataProps`?